kitsune/sumo/utils.py

```diff
@@ -328,3 +329,16 @@ def is_ratelimited(request, name, rate, method=['POST'], skip_if=lambda r: False
             Record.objects.info('sumo.ratelimit', '{key} hit the rate limit for {name}',
                                 key=key, name=name)
     return request.limited
+
+
+def get_browser(request):
```

Comment:
Speaking of tests, utility functions that are "pure" (have no side effects) like this one are prime candidates for testing. They are often easy to test (when made well), and since other code relies on them to work as expected, it is vital to ensure that they work using tests.